.github/workflows/playwright.yml

```diff
@@ -10,14 +10,17 @@ jobs:
     timeout-minutes: 60
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v3
+      - name: Remove all fonts
```

Comment:
I'm surprised latin text renders properly, but I guess it's included in mincho...?

Reply:
Yes, it is.